refactor(integrations): apply PostgreSQL pattern to every integration article (#920)
microsoft/aspire.dev: src/frontend/src/content/docs/integrations/caching/redis/redis-get-started.mdx

---
title: Get started with the Redis integrations

> description: Understand how the Aspire Redis integrations fit together — model a Redis resource in your AppHost, then connect to it from any consuming app.

Reviewer: In this article, you don't do the modelling or connection.
```suggestion
description: Understand why you use the Aspire Redis integrations and how they fit together.
```
Please also address this same issue in the get started articles for all the other integrations in this PR.

Author: @alistairmatthews Applied across the board in `fb08de46`. Every get-started article in this PR now uses: `description: Understand why you use the Aspire {X} integrations and how {they fit|it fits} together.`

---

import { Image } from 'astro:assets';
import { LinkButton, Steps } from '@astrojs/starlight/components';
import redisIcon from '@assets/icons/redis-icon.png';

<Image
  src={redisIcon}
  alt="Redis logo"
  width={100}
  height={100}
  class:list={'float-inline-left icon'}
  data-zoom-off
/>

> [Redis®](#registered-trademark) is the original RESP-compatible (Redis Serialization Protocol) in-memory data store — a high-performance key/value cache, message queue, and primary data platform used by millions of applications worldwide. The Aspire Redis integration lets you model a Redis server as a first-class resource in your AppHost, then hand the connection information to any consuming app — regardless of language. Valkey and Garnet are drop-in alternatives that speak the same protocol and work with the same client libraries.

Reviewer: The first clause in the first sentence seems a bit circular. Please replace with:
```suggestion
[Redis®](#registered-trademark) is the world's fastest data platform for caching, vector search, and NoSQL databases — a high-performance key/value cache, message queue, and primary data platform used by millions of applications worldwide. The Aspire Redis integration lets you model a Redis server as a first-class resource in your AppHost, then hand the connection information to any consuming app — regardless of language. Valkey and Garnet are drop-in alternatives that speak the same protocol and work with the same client libraries.
```

Author: @alistairmatthews Adopted your suggestion verbatim in `fb08de46`. Thanks!

## How the pieces fit together

The Redis integration has two sides: a **hosting integration** that you use in your AppHost to model the Redis resource, and a **connection story** for consuming apps that reference it.

```mermaid
architecture-beta

  group apphost(server)[AppHost]
  group consumer(server)[Consuming app]

  service hosting(server)[Hosting integration] in apphost
  service redis(server)[Redis server] in apphost

  service client(iconoir:server-connection)[Client integration] in consumer
  service app(server)[App] in consumer

  hosting:R --> L:redis
  redis:R --> L:client
  client:R --> L:app
```

The **hosting integration** lives in your AppHost project and models the Redis server as a resource. The **client integration** lives in each consuming app and uses the connection information Aspire injects to talk to Redis.

Getting there is a two-step process: model the Redis resource in your AppHost, then connect to it from each app that needs it.

<Steps>

1. ### Model Redis in your AppHost

>     Add the Redis hosting integration to your AppHost, then declare a Redis resource and reference it from the apps that need it. The [Redis Hosting integration](/integrations/caching/redis/redis-host/) reference walks through every capability — data volumes, data bind mounts, persistence snapshots, Redis Insight, Redis Commander, and custom parameters — with side-by-side C# and TypeScript examples.

Reviewer: Using "reference" in this sentence might be confusing. Instead use: 
```suggestion
    Add the Redis hosting integration to your AppHost, then declare a Redis resource and reference it from the apps that need it. The [Redis Hosting integration](/integrations/caching/redis/redis-host/) article walks through every capability — data volumes, data bind mounts, persistence snapshots, Redis Insight, Redis Commander, and custom parameters — with side-by-side C# and TypeScript examples.
```
Please also address this same issue in the get started articles for all the other integrations in this PR.

Author: @alistairmatthews Adopted your suggestion in `fb08de46`, and rolled out the "article walks through" wording (replacing "reference walks through") across all 55 get-started articles in this PR.
